docs/reference/release-notes/850.md

```diff
@@ -0,0 +1,188 @@
+
+### Zeebe REST API infrastructure to build user task API
+
+The first step to offer an intuitive and consistent experience via a single, [unified Camunda 8 REST API](https://camunda.com/blog/2024/03/streamlining-camunda-apis-zeebe-rest-api/) is to provide Zeebe REST API. With this release, developers can use Zeebe REST API to manage [Zeebe User Tasks](/apis-tools/tasklist-api-rest/migrate-to-zeebe-user-tasks.md), enabling immediate task state changes. Zeebe REST API includes support for identity authentication and multi-tenancy, ensuring parity to Zeebe gRPC API.
+
+### Refactoring Suggestions
+
+Introducing AI-driven Refactoring Suggestions in our modeling canvas, designed to refine your BPMN models based on task descriptions. Enter a description, like 'Send a message', and immediately receive a practical suggestion, such as implementing it through a Slack connector. This feature aims to streamline your workflow, offering intuitive, actionable advice for transforming descriptions into efficient BPMN tasks.
+
+### Frontend Developer Documentation
+
+Developers now find [documentation](/apis-tools/frontend-development/01-task-applications/01-introduction-to-task-applications.md) about how to build custom frontends using Camunda's APIs and tools, including form-js.
+
+### KPI Report Wizard Framework
+
+The KPI Wizard is the new way to build a KPI Report quickly. The Wizard already contains predefined KPIs that can be selected. It guides the users through the needed customization and allows them to activate an alert that allows passive monitoring. The list of predefined KPIs will be extended with upcoming releases. Start setting up a first Process KPI now.
+
+### Use Secrets in Hybrid Mode
+
+You are now able to consume the secrets from the console in your own custom connector which is connected to our SAAS platform.
+
+### Multi-Region: Productise the 2 Regions, Active-Active, Self-Managed setup (Zeebe, Operate, Tasklist)
+
+You can now run Zeebe, Operate and Tasklist across 2 geographical regions. You will find installation and failover guides to help you recover quickly and safely from whole region failures.
+
+Note that there is no dedicated guides how to setup Optimize, Identity and Connectors across regions yet.
+
+### REST Postman Collection to Connector Template Converter
+
+Via a command line tool you can now generate your connector template based on our REST Connector from a postman collection. We are supporting multiple operations + splitting up the body to single fields
+
+### SOAP SDK Support (SM + Hybrid)
+
+You can now connect to a SOAP service via our Camunda Connector. The first iteration of this connector will be only available in SM and as hybrid setup
+
+### Process Blueprint Import via Link
+
+You can directly use process blueprints from our marketplace in your SAAS Web Modeler. Just click on "For SAAS". The blueprint will be downloaded to your web modeler and is ready to use.
```

Comment:
Is the UI really written out as "For SAAS" or "For SaaS"?

Reply:
Good catch, not sure.

Reply:
<img width="1563" alt="image" src="https://github.com/camunda/camunda-docs/assets/1051363/b9d14511-c688-481c-b650-628cd21ba70c">

Confirmed, **For SAAS**